test/mongo-dao.ts

```diff
@@ -18,3 +18,13 @@ describe(`Mongo dao`, () => {
       {thumbnail: 0, doi: 0, publisher: 1, title: 1})
   })
 })
+
+describe('Waiting for db connection', function() {
+    it('should wait awhile before continuing', function(done) {
+        this.timeout(61000); // Optional: set timeout for this test to 61 seconds
+        setTimeout(function() {
+            // Your test logic here
```

Comment:
currently a no-op test?

Reply:
True. But it creates a 10 second delay which might not be very elegant. But I had to make sure that the oai-pmh-service part had a chance to connect to the mongodb server. Otherwise all tests fail.